querydsl-examples/querydsl-example-sql-guice/pom.xml

```diff
@@ -92,8 +92,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
```

Comment:
Should this be 8, rather than 1.8, as it is below in querydsl-example-sql-spring/pom.xml?

Reply:
Yes, thanks for pointing it out!